### ISSUE TITLE
Advanced planet configuration - biome settings

### ISSUE THREAD

**Issue by donqixot** [zmaster587/AdvancedRocketry#932]
Minecraft Version:
-1.11.2
Forge Version:
-13.20.1.2530
LibVulpes Version:
-1.11.2-0.2.6
*is this the latest curseforge build?* yes
----------
Explain issue in detail:
I am trying to use advanced planet configuration to specify biome from Galacticraft (Mars Flat). Biome created is correct, but terrain cover is all grass. Galacticraft has grass/dirt/stone replaced with their version of blocks. Is there any way to either let Galacticraft create my custom dimension on it's own or specify what blocks to use for grass/dirt/stone like BiomeTweaker does?

Basically, I am trying to create my custom version of Mars similar to Galacticraft, but with all the benefits from your mod, which I love very much and for which I am very grateful.

![2017-11-08_22 00 34](https://user-images.githubusercontent.com/12219588/32586886-50edfae0-c4d3-11e7-9612-7be1d825404e.png)

**Comment by zmaster587** [comment]
Galacticcraft I believe has their own terrain generator which I would either have to use (which would make the rest of the biomes not work right on that planet) or copy (legality depending on Galacticraft's licence).

You can however just let galacticraft create its planets then using dimMapping in the xml to map galacticraft's dimension to an Advanced rocketry planet.  If you like there's also a config in AR that (in theory) disabled galacticraft's Oxygen system if you don't want to have to deal with AR's and GC's at the same time.

**Comment by donqixot** [comment]
Cool, thanks! I will try all that! Close for now.
